physionet-django/export/views.py

```diff
@@ -8,6 +8,9 @@
 from rest_framework import mixins
 from rest_framework.response import Response
 
+# Importing the get_content function from Search Module's views.py
```

Comment:
I don't think we need this comment (it is clear from the import what is happening)

Reply:
I use these comments as a guide for writing the code. 
I will ensure to remove simple ones later on.